Dockerfile-csi-test

```diff
@@ -14,6 +14,8 @@
 
 FROM golang:1.11
 
+ARG CSI_PARAMS
```

Comment:
consider adding a default value

Reply:
already thought about it, no reason to add default since every storage array has its own param

Reply:
how would the ds8k param look? must it be different from both existing param files?

Reply:
as a9k AFAIK, no issues on that as was on svc

Reply:
then I think having a non-svc default is better than nothing